frontend/console/src/components/List.tsx

```diff
@@ -0,0 +1,24 @@
+import { classNames } from '../utils'
+
+type ListProps<T> = {
+  items: T[]
+  renderItem: (item: T) => React.ReactNode
+  onClick?: (item: T) => void
+  className?: string
+}
+
+export const List = <T,>({ items, renderItem, onClick, className }: ListProps<T>) => {
```

Comment:
Did you mean to remove this `,` or is that doing some fancy JS magic I haven't learned yet? :)

Reply:
I think the `,` is required for the component to parse correctly. Removing it causes all kinds of errors 😂 . I think it's just a react generic component feature.

Reply:
oh wow that's good to know!